source/Calamari.Shared/Integration/Certificates/WindowsNative/CertificatePal.cs

```diff
@@ -115,6 +114,8 @@ public static byte[] GetCspPrivateKeySecurity(SafeCspHandle cspHandle)
             return buffer;
         }
 
+        // At the moment these models are not in the Calamari Solution. This is the next step.
+#if WINDOWS_CERTIFICATE_STORE_SUPPORT
         public static byte[] GetCngPrivateKeySecurity(SafeNCryptKeyHandle hObject)
```

Comment:
The `SafeNCryptKeyHandle` structures are not currently defined in the Octopus solution like some of the others, and so this will need to be introduced as netcore does not have access to them from mscorlib like natframework does.

To keep the changes in multiple stages, this will be introduced in a followup PR